packages/preact/src/index.ts

```diff
@@ -358,12 +358,39 @@ export function useComputed<T>(compute: () => T) {
 	return useMemo(() => computed<T>(() => $compute.current()), []);
 }
 
+let oldNotify: (this: Effect) => void,
+	queue: Array<Effect> = [],
+	isFlushing = false;
+
+function flush() {
+	batch(() => {
+		let flushing = [...queue];
+		isFlushing = false;
+		queue.length = 0;
+		for (let i = 0; i < flushing.length; i++) {
+			oldNotify.call(flushing[i]);
+		}
+	});
+}
+
+function notify(this: Effect) {
+	queue.push(this);
+	if (!isFlushing) {
+		isFlushing = true;
+		(typeof window === "undefined" ? setTimeout : requestAnimationFrame)(flush);
```

Comment:
Should it fallback to `queueMicrotask` to preserve old behavior in environments that doesn't have window object? (React Native)

Reply:
Window should be defined in React native and so should `requestAnimationFrame` iirc, RN does something like `global.window = global`

Reply:
I am pretty much sure there is no window. I will recheck. But there is requestAnimationFrame for sure